test/built-ins/Array/prototype/includes/from-index-default.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2014 Robert Kowalski. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+    Array.prototype.includes should search the whole array, as the optional second argument fromIndex defaults to 0
+author: Robert Kowalski
```

Comment:
Update: I reviewed the CLA responses and there is no "Robert Kowalski". Please have the author sign http://tc39.github.io/test262-cla/